style.css

```diff
@@ -222,6 +222,20 @@ h6,
   margin-top: 0;
   margin-bottom: 1.5rem;
 }
+h1#related-topics,
+h2#related-topics,
+h3#related-topics,
+h4#related-topics,
+h5#related-topics,
+h6#related-topics,
+.h1#related-topics,
+.h2#related-topics,
+.h3#related-topics,
+.h4#related-topics,
+.h5#related-topics,
+.h6#related-topics {
+  margin-top: 1rem;
+}
```

Comment:
This seems to be excessive. Do we really use all these 12(6) for related topic headers?